partners/anaconda/base/anaconda-ce-imagestream.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: s2i-minimal-notebook-anaconda
+  labels:
+    opendatahub.io/notebook-image: "false"
+    opendatahub.io/modified: "false"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/Xaenalt/s2i-minimal-notebook"
```

Comment:
should this be your private github or should it be in an official org somewhere?

Reply:
That can get changed once the changes actually get merged to an official org xD

Reply:
@Xaenalt Is there a PR for "the changes"?

Reply:
n/m, found it:  https://github.com/red-hat-data-services/s2i-minimal-notebook/pull/1